stack-snapshot.yaml

```diff
@@ -14,9 +14,7 @@ packages:
   - ghc-lib-parser-ex-8.8.5.8
   - bytestring-nums-0.3.6
   - extra-1.7.1
-  - github: awakesecurity/gRPC-haskell
```

Comment:
we should revert this file. It doesn’t break anything in your PR because you didn’t repin (you could via `bazel run @stackage-unpinned//:pin`) as well.

Reply:
aha! thanks for pointing this out, it's reverted now